plugin/commentary.vim

```diff
@@ -87,6 +126,12 @@ if !hasmapto('<Plug>Commentary') || maparg('gc','n') ==# ''
   nmap gcc <Plug>CommentaryLine
   nmap cgc <Plug>ChangeCommentary
   nmap gcu <Plug>Commentary<Plug>Commentary
+  xmap gcy   <Plug>CommentaryYank
+  nmap gcy   <Plug>CommentaryYank
+  nmap gcyy  <Plug>CommentaryYankLine
+  xmap gcd   <Plug>CommentaryDupe
+  nmap gcd   <Plug>CommentaryDupe
```

Comment:
Won't this cause a pause since `<Plug>CommentaryDupeLine` overlaps `<Plug>CommentaryDupe`? (solution would be to wrap in parentheses, e.g., `<Plug>(CommentaryDupeLine)` and `<Plug>(CommentaryDupe)`)